sklearn/preprocessing/data.py

```diff
@@ -2873,3 +2873,39 @@ def inverse_transform(self, X):
                     X_tr[mask, idx] = None
 
         return X_tr
+
+def get_feature_names(self, input_features=None):
```

Comment:
this should be defined as a method within the class, not as an external function